src/test/java/core/basesyntax/FruitServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Storage;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.AdditionStrategy;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.FruitShopServiceImpl;
+import core.basesyntax.service.Operation;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.ReduceStrategy;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static List<FruitRecordDto> expectedTransactionDtos;
+    private static FruitShopService fruitService;
+
+    @After
+    public void clear() {
+        Storage.fruits.clear();
+    }
+
+    @BeforeClass
+    public static void setUp() {
+        Map<Operation, OperationStrategy> operationStrategyMap = new HashMap<>();
+        operationStrategyMap.put(Operation.BALANCE, new AdditionStrategy());
+        operationStrategyMap.put(Operation.SUPPLY, new AdditionStrategy());
+        operationStrategyMap.put(Operation.RETURN, new AdditionStrategy());
```

Comment:
maybe create local variable for new AdditionStrategy(), and use variable

Reply:
done